news/2021-08-19-project-loved-august-2021.md

```diff
@@ -0,0 +1,283 @@
+
+A very accessible mapset with a multitude of difficulties, the main selling point of *Future Cider* is its energetic nature expressed in a clean and fun manner. Lower difficulties are a great introduction to light chordstreaming with additional bursts and some simple long notes. As the difficulty progresses, more and more complex patterns appear, accompanied with streamy LNs with increasing density. The top 3 difficulties, including Jakads' and Bpple's collab Blue Another and two iterations of Future Candy feature a whole new level of challenge with stutter and bump SVs that are present during virtually the whole chart. Futuristic Candy is the top difficulty, and it builds on Future Candy with LN choruses being even more difficult. Even though the harder charts deliver lots of diverse patterns to process, they aren't overdone and greatly emphasize the song.
+
+This set has waited way too long in the darkness and we're glad to finally bring one of the most popular 7K sets to the Loved voting.
+
+[![](/wiki/shared/news/2021-08-19-project-loved-august-2021/351345.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Randy Mortimer - Penguin (Pinnacle Remix) (Cut Ver.)](https://osu.ppy.sh/beatmapsets/351345#mania) by [Cuppp](https://osu.ppy.sh/users/2345293) and [Onta_Metalslug](https://osu.ppy.sh/users/3518460)\
+170 BPM, 1:36 | 6.54★, 11.09★\
+*description written by [Penguinosity](https://osu.ppy.sh/users/10235296)*
+
+Cuppp and gemboyong are here to bring their notorious maps of *Penguin* to Loved. Neither mapper should need much of an introduction as both Cuppp and gemboyong have had entries in Loved for a long time now, but I'll at least provide a bit of history to these two maps for any who are unaware, starting with Cuppp's 7K diff.
+
+This map is already near 6 years old but still a hilarious demonstration of just how far you can take the O2Jam-styled patterning for a song like this. The intro to the chart is pretty tame, but once it's over, you're quickly thrown into what I can only describe as "LN Art". Full LN double stairs moving from left to right, uncomfortable anchored LN holds and gaps, and on top of that, almost all of this is charted to 1/8. For the map being just over a minute thirty, Cuppp managed to pack in more pain and suffering than you'll see in most maps of this vein more than triple its length. Now onto gemboyong's 4K difficulty, they used Cuppp's map as the blueprint and manually converted any of the patterns that could comfortably fit with the lesser note count to 4K. Many 4K players should be aware of this diff specifically since it's featured in their popular [Gems pack 13](https://osu.ppy.sh/beatmapsets/555819) that was uploaded in 2017.
+
+Anyways, I'm glad we were able to get gemboyong's difficulty included, and the Loved team is happy to be finally bring you these masterpieces. LET'S GO!
```

Comment:
always copy-paste the whole line if you have to manually apply changes

```suggestion
Anyways, I'm glad we were able to get gemboyong's difficulty included, and the Loved team is happy to finally bring you these masterpieces. LET'S GO!
```

Reply:
whoops, will be fixed when given the all-clear for forum posts